src/strings/PowerFxResources.en-US.resx

```diff
@@ -4760,4 +4760,10 @@
     <value>Unsupported untyped/JSON conversion type '{0}' in argument.</value>
     <comment>Error Message shown to user when a unsupported type is passed in type argument of AsType, IsType and ParseJSON functions.</comment>
   </data>
+  <data name="ErrReachedMaxJsonDepth" xml:space="preserve">
+    <value>Maximum depth reached while traversing JSON payload.</value>    
+  </data>
+  <data name="ErrReachedMaxJsonLength" xml:space="preserve">
+    <value>Maximum length reached in JSON function.</value>    
```

Comment:
>JSON [](http://example.com/codeflow?start=37&length=4)

Please add a comment with a `{Locked=JSON}` directive -since this is a function name we don't want it translated in case some language has a different acronym for that.